.spelling

```diff
@@ -330,6 +330,7 @@ docker.io
 dogfood
 double-tls
 Drucker
+Dual-stack
```

Comment:
Maybe lower case as I think we should lower case in the table (if not remove the `-` and then this entry isn't needed). I didn't see an upper case instance.